manual/cops_style.md

```diff
@@ -5741,6 +5741,42 @@ question = '"What did you say?"'
 
 * [https://rubystyle.guide#percent-q](https://rubystyle.guide#percent-q)
 
+## Style/RedundantRegexpEscape
+
+Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+--- | --- | --- | --- | ---
+Enabled | Yes | Yes  | 0.83 | -
```

Comment:
Looks like the file needs to be generated again, looking at the "version added".

Reply:
Oops, yes, good catch, I missed this after I merged master and bumped the `VersionAdded` you noted above - I'll regenerate now and can change again when @bbatsov advises about the next version number